src/main/java/ch/njol/skript/classes/data/BukkitClasses.java

```diff
@@ -362,7 +362,8 @@ protected boolean canBeInstantiated() {
 				.description("A location in a <a href='#world'>world</a>. Locations are world-specific and even store a <a href='#direction'>direction</a>, " +
 						"e.g. if you save a location and later teleport to it you will face the exact same direction you did when you saved the location.")
 				.usage("")
-				.examples("")
+				.examples("teleport player to location at 0, 69, 0",
```

Comment:
i'd like to see a slightly different example rather than a repetition of the same one twice. Maybe throw in a use of the location function, or use pitch/yaw, or use a different effect.

Reply:
I wanted to see the first example really simple. Okay, I changed the second example.